### PR TITLE
Fix missing file termination

### PR DESCRIPTION
**Pull request by michael-o**
Upstream patch for Debian's 21_fix_conf_file.patch

### CODE REVIEW

**Comment by michael-o** [comment]
Indeed, it is missing everywhere. I'll pick it up and change the PR.

File: include/distconfig.php [hunk 1]
```diff
@@ -493,3 +493,4 @@
 // $config->expandTabsBy(3, 'myrep'); // Expand Tabs by 3 for repository 'myrep'
 
 // }}}
+?>
```
Comment (joubertredrat) [inline include/distconfig.php]: We don't close PHP tags on pure php files

**Comment by tschoening** [comment]
RTFM:

> If a file is pure PHP code, it is preferable to omit the PHP closing tag at the end of the file.

http://php.net/manual/en/language.basic-syntax.phptags.php

Is there any hint on the intent of the Debian fix? Which problem should it fix?

**Comment by michael-o** [comment]
Thanks for the information guys, I wasn't aware of. Don't do PHP day by day.

**Comment by tschoening** [comment]
Me neither and to be honest I would have expected that the Debian guys already checked those things. :-)

**Pull request closed without merging**